app-misc/figlet/figlet-222.recipe

```diff
@@ -9,24 +9,39 @@ COPYRIGHT="1991, 1993, 1994 Glenn Chappell and Ian Chai
 	2002 Christiaan Keet"
 LICENSE="AFL v2.1"
 REVISION="1"
-SOURCE_URI="ftp://ftp.figlet.org/pub/figlet/program/unix/figlet222.tar.gz"
-CHECKSUM_MD5="5f84ad52d092d5db4ad5916df767646b"
-STATUS_HAIKU="stable"
-DEPEND=""
+SOURCE_URI="http://ftp.nsysu.edu.tw/FreeBSD/ports/local-distfiles/obrien/figlet222.tar.gz"
+CHECKSUM_SHA256="21f8f9e9b2190fbc0f801fb4704355e5d73d36c55670272cd08cad3408d1f5fe"
+SOURCE_DIR="figlet222"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+PROVIDES="
+	figlet = $portVersion
+	app:Figlet =$portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
 
 BUILD()
 {
-	cd figlet222
 	make
 }
 
 INSTALL()
 {
-	cd figlet222
 	DISTRODIR=${DESTDIR}
-	MANDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet/share/man
-	DEFAULTFONTDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet/fonts
-	DESTDIR=${DISTRODIR}/`finddir B_APPS_DIRECTORY`/figlet
+	MANDIR=${DISTRODIR}/$binDir/figlet/share/man
```

Comment:
try:
MANDIR=$manDir
this should result in the man file(s) ending up in the proper location in documentation/man/man6